cocos/core/root.ts

```diff
@@ -312,6 +310,12 @@ export class Root {
         return true;
     }
 
+    public createDefaultPipeline () {
+        const rppl = new legacyCC.ForwardPipeline();
```

Comment:
why use legacyCC.ForwardPipeline ? for native compatibility ?

I think if you import FrowardPipeline from `renderer` module, it can be automatically switched to native ForwardPipeline

Reply:
> why use legacyCC.ForwardPipeline ? for native compatibility ?
> 
> I think if you import FrowardPipeline from `renderer` module, it can be automatically switched to native ForwardPipeline

modified